optimum/exporters/onnx/model_configs.py

```diff
@@ -1182,6 +1182,97 @@ def outputs(self) -> Dict[str, Dict[int, str]]:
         }
 
 
+class T5EncoderOnnxConfig(CLIPTextOnnxConfig):
```

Comment:
why is inheriting from `CLIPTextOnnxConfig` needed ?

Reply:
no need, fixed